pkg/thing/interactors/update_data.go

```diff
@@ -0,0 +1,69 @@
+package interactors
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+)
+
+var (
+	ErrNoSchema = fmt.Errorf("thing has no schema")
+	ErrDataInvalid = fmt.Errorf("data is incompatible with thing's schema")
+)
```

Comment:
Two small changes are needed here :grimacing: 

1. Exported errors should be annotated;
2. They must be aligned for easier readability;

I recommend you use [this extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.Go) so that `go fmt` run on-save (for aligning, for example) and `go lint` to point out anything we missed on (such as annotating exported identifiers) :smile: 

Here's a quick suggestion :point_down: 
```suggestion
var (
	// ErrNoSchema is returned when a schema is empty.
	ErrNoSchema = fmt.Errorf("thing has no schema")

	// ErrDataInvalid is returned when the data's and thing's valueType mismatch.
	ErrDataInvalid = fmt.Errorf("data is incompatible with thing's schema")
)
```

Reply:
@lcbm Awesome! I've install this extension and made the changes, done! :+1: